### PR TITLE
add note about _.minBy and _.maxBy to readme

### PR DESCRIPTION
**Pull request by alexesDev**

### CODE REVIEW

**Comment by cht8687** [comment]
LGTM... @stevemao  any more comments?

File: README.md [hunk 1]
```diff
@@ -658,6 +659,59 @@ Tests whether some element in the array passes the test implemented by the provi
...
+  ```js
+  // utils
+  const makeSelect = (comparator) => (a, b) => comparator(a, b) ? a : b
+  const minByValue = makeSelect((a, b) => a.value <= b.value)
+  const maxByValue = makeSelect((a, b) => a.value <= b.value)
+
+  // main logic
+  const data = [{ value: 6 }, { value: 2 }, { value: 4 }]
+  const minItem = data.reduce(minByValue, {})
+  const maxItem = data.reduce(minByValue, {})
```
Comment (stevemao) [inline README.md]: `data.reduce(maxByValue, {})`?

Comment (cht8687) [inline README.md]: seems a typo here...

 should we create a ESlint rule for _.minBy _.maxBy after this merge? @stevemao

Comment (stevemao) [inline README.md]: Currently the eslint plugin only checks methods that can be 1 to 1 mapped. I feel including these in the plugin might be too complicated.

**Pull request closed without merging**